process/using_copykit_artifacts/README.md

```diff
@@ -0,0 +1,121 @@
+# Getting Prototype Pages Into App
+
+This is a Guide to using the files/artefacts created by copykit
+
+For details of copykit itself see
+
+* CopyKit repo : <https://github.com/EnvironmentAgency/copykit>
+* [copykit usage](/process/using-copykit)
+
+## Steps
+
+Copykit will produce a set of Form Objects (to work with Reform gem) per state/page.
+
+Assuming you are using an Enrollment model with a state machine,
+then your EnrollmentsController needs to be decorated with following snippet to setup
+the appropriate Form Object on a state by state basis
+
+```ruby
+DigitalServicesCore::EnrollmentsController.class_eval do
+  before_filter :setup_form_object
+  private
+  def setup_form_object
+    @enrollment ||= DigitalServicesCore::Enrollment.new
+    @form = "States::#{@enrollment.state.classify}State::FormBuilder".constantize.call(@enrollment)
+  end
+```
+
+Reform does not seem to take automatic care of param whitelisting. So check whether EnrollmentsController is setup
+ to use StrongParameters style of white-listing inbound params and add in new params.
+
+Good example in WasteExemptionsShared repo, under  :
```

Comment:
Don't bother with colons at the end of lead ins to examples. Again trying to settle on a convention and feel that having nothing might be like being pelted with night soil for grammatical purists, but is actually cleaner for the reader and easier for the writer.

N.B. Promise I will get round to documenting all these decisions in the near future 😁 !